### ISSUE TITLE
DAILY GRAPH DOES NOT SHOW DATA ON RECOVERED AND DECEASED

### ISSUE THREAD

**Issue by lol-cod** [nic-delhi/AarogyaSetu_Android#257]
![IMG-20200528-WA0003](https://user-images.githubusercontent.com/66051409/83105977-4d183180-a0d9-11ea-9abd-285912d0f178.jpg)
![IMG-20200528-WA0004](https://user-images.githubusercontent.com/66051409/83105999-55706c80-a0d9-11ea-86be-b144fec33256.jpg)

>

**Comment by aravindvnair99** [comment]
@lol-cod Unable to reproduce your issue. The graph seems to be working perfectly fine for me. Try restarting the app or your phone. You can also try clearing your cache. Does that help?

**Comment by malayaku** [comment]
same issue in my Honor mobile

**Comment by aravindvnair99** [comment]
> same issue in my Honor mobile

@malayaku Which version is your app?

**Comment by malayaku** [comment]
Aarogya Setu App - v1.13
Android Ver - 7.0
Mobile - Honor 6x

**Comment by aravindvnair99** [comment]
> Aarogya Setu App - v1.13
> Android Ver - 7.0
> Mobile - Honor 6x

@malayaku Try restarting the app or your phone. You can also try clearing your cache. Does that help? Please provide steps to reproduce the issue.

**Comment by aravindvnair99** [comment]
> Aarogya Setu App - v1.13
> Android Ver - 7.0
> Mobile - Honor 6x

I assume you meant v1.1.3

**Comment by malayaku** [comment]
yes  v1.1.3

**Comment by malayaku** [comment]
even after clearing cache - issue still exist

Select "Daily" (not cumulative) & go to recovered & deceased section. Issue will be seen

**Comment by aravindvnair99** [comment]
> even after clearing cache - issue still exist
> 
> Select "Daily" (not cumulative) & go to recovered & deceased section. Issue will be seen

@malayaku Strange, I'm unable to reproduce your issue. Restart your phone, try again. This time, take a logcat. Perhaps the logs can shed some light. Attach the log here.

**Comment by lol-cod** [comment]
See graph on daily 
You will be able to see it

**Comment by lol-cod** [comment]
@aravindvnair99  It's a bug in new version I think which version are you using

**Comment by ArjitRout** [comment]
I am also facing same issue when selecting "Daily" and go to recovered & deceased section. Issue will be seen

**Comment by aravindvnair99** [comment]
> @aravindvnair99  It's a bug in new version I think which version are you using
> 

@lol-cod I'm on v1.1.3 and I don't face any issues. That's the latest version which was released on 26th May 2020. Which version are you on?

**Comment by aravindvnair99** [comment]
> I am also facing same issue when selecting "Daily" and go to recovered & deceased section. Issue will be seen

@ArjitRout Did you try clearing cache or restarting? Which phone do you have, app version and Android version?

**Comment by lol-cod** [comment]
@aravindvnair99 I am on v1. 1.13
The latest one
Android version6. 0.1

**Comment by aravindvnair99** [comment]
> @aravindvnair99 I am on v1. 1.13
> The latest one
> Android version6. 0.1

@lol-cod Okay, I'm on Android 10. Check logcat using ADB. Perhaps that might shed some light.

**Comment by lol-cod** [comment]
@aravindvnair99 it's still the same

**Comment by aravindvnair99** [comment]
> @aravindvnair99 it's still the same

@lol-cod Show me your log file. If you are not familiar with logcat, have a look at [this](https://www.xda-developers.com/guide-sending-a-logcat-to-help-debug-your-favorite-app/).

@ArjitRout @malayaku you both can do the same as well.

**Comment by ArjitRout** [comment]
@aravindvnair99 **still same issue exists:-** After clearing cache and one time uninstalled and installed again also.
Realme 2 pro
App version:- 1.1.3v
Android version:- 9

**Comment by ArjitRout** [comment]
> > @aravindvnair99 it's still the same
> 
> @lol-cod Show me your log file. If you are not familiar with logcat, have a look at [this](https://www.xda-developers.com/guide-sending-a-logcat-to-help-debug-your-favorite-app/).
> 
> @ArjitRout @malayaku you both can do the same as well.

Sending log

**Comment by ArjitRout** [comment]
Need a apk file to run in android studio and get the log

**Comment by aravindvnair99** [comment]
> Need arogya setu app .apk file to run in android studio and get the log

@ArjitRout No, you can take the log from your phone itself. Android Studio isn't required. Follow [this](https://www.xda-developers.com/guide-sending-a-logcat-to-help-debug-your-favorite-app/)
